src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: "Hands-on with Portals: seamless navigation on the Web"
 subhead: |
-  Learn how to build good navigation UX with the Portals API.
+  Learn how the proposed Portals API could change the way we build good navigation UX.
 date: 2019-05-06
 updated: 2019-08-29
 authors:
   - uskay
 hero: hero.png
 alt: A logo of Portals
 description: |
-  Portals help keep your front-end simple while allowing seamless navigations
+  A newly proposed Portals API helps keep your front-end simple while allowing seamless navigations
```

Comment:
```suggestion
  The newly proposed Portals API helps keep your front-end simple while allowing seamless navigations
```
Definite article since there's only one Portals API.